src/Interpreters/Streaming/Aggregator.h

```diff
@@ -74,15 +76,11 @@ namespace Streaming
   *  best suited for different cases, and this approach is just one of them, chosen for a combination of reasons.
   */
 
-enum class ConvertAction : uint8_t
+enum class AggregateStateType
```

Comment:
Once we have `std::optional<TrackingUpdatesType>`, do we still need this ?

Reply:
TrackingUpdatesType indicates that the aggregate states requires tracking updates. But here we need to specify what content we want to convert (All or OnlyUpdates, or OnlyRetract)

I will refine the AggregateStateType  to ConvertType :

enum class ConvertType : uint8_t
{
    Normal = 0,
    OnlyUpdates = 1,
    OnlyRetract = 2,
};